src/main/java/emu/grasscutter/data/excels/OpenStateData.java

```diff
@@ -1,13 +1,58 @@
-package emu.grasscutter.game.props;
+package emu.grasscutter.data.excels;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.stream.Stream;
+import emu.grasscutter.data.GameData;
+import emu.grasscutter.data.GameResource;
+import emu.grasscutter.data.ResourceType;
+import lombok.AccessLevel;
+import lombok.Getter;
 
-import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
-import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
 
-public enum OpenState {
+@ResourceType(name = "OpenStateConfigData.json", loadPriority = ResourceType.LoadPriority.HIGHEST)
+public class OpenStateData extends GameResource {
+    public static String PLAYER_LEVEL_UNLOCK_COND = "OPEN_STATE_COND_PLAYER_LEVEL";
+    public static String QUEST_UNLOCK_COND = "OPEN_STATE_COND_QUEST";
+    public static String OFFERING_LEVEL_UNLOCK_COND = "OPEN_STATE_OFFERING_LEVEL";
+    public static String REPUTATION_LEVEL_UNLOCK_COND = "OPEN_STATE_CITY_REPUTATION_LEVEL";
+    public static String PARENT_QUEST_UNLOCK_COND = "OPEN_STATE_COND_PARENT_QUEST";
+
+    private int id;
+    @Getter private boolean defaultState;
+    @Getter private boolean allowClientOpen;
+    @Getter private int systemOpenUiId;
+    @Getter private List<OpenStateCond> cond;
+
+    public static class OpenStateCond {
+        @Getter private String condType;
```

Comment:
`condType` should be an enum instead of String. Gson automatically parses them to an enum.